src/coreclr/tools/aot/ILCompiler/WasmConfigurablePInvokePolicy.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using Internal.IL.Stubs;
+using Internal.TypeSystem;
+
+namespace ILCompiler
+{
+    internal sealed class WasmConfigurablePInvokePolicy : ConfigurablePInvokePolicy
+    {
+        public WasmConfigurablePInvokePolicy(TargetDetails target, IReadOnlyList<string> directPInvokes, IReadOnlyList<string> directPInvokeLists)
+            : base(target, directPInvokes, directPInvokeLists)
+        {
+        }
+
+        public override bool GenerateDirectCall(MethodDesc method, out string externName)
+        {
+            if (IsWasmImport(method, out PInvokeMetadata pInvokeMetadata))
+            {
+                externName = "__wasm_import_" + pInvokeMetadata.Module + "_" + pInvokeMetadata.Name;
```

Comment:
> pInvokeMetadata.Name

Does this work correctly with PIs that don't specify `EntryPoint` (if not, it also needs a test)?

Reply:
It seems to yes, added the test:
```cs
private static void TestNamedModuleCallWithoutEntryPoint()
{
    StartTest("Wasm import from named module test");
    EndTest(ModuleFuncWithoutEntryPoint(77) == 77);
}

[DllImport("ModuleName"), WasmImportLinkage]
private static extern int ModuleFuncWithoutEntryPoint(int x);
```

Generates
```llvm
declare i32 @__wasm_import_ModuleName_ModuleFuncWithoutEntryPoint(i32) #4

define ptr @get.__wasm_import_ModuleName_ModuleFuncWithoutEntryPoint() {
GetExternFunc:
  ret ptr @__wasm_import_ModuleName_ModuleFuncWithoutEntryPoint
}

attributes #4 = { "wasm-import-module"="ModuleName" "wasm-import-name"="ModuleFuncWithoutEntryPoint" }
```